CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## 3.0.0 (Unreleased)
 DEPRECATIONS:
 * resource_sumologic_ingest_budget : Deprecated in favour of `resource_sumologic_ingest_budget_v2`.
+BUG FIXES:
```

Comment:
This is bug fix and we can be released as part of 2.31.5

Reply:
done